src/domains/chart/sagas.ts

```diff
@@ -241,38 +241,53 @@ function* fetchChartSaga({ payload }: Action<FetchChartPayload>) {
 function* fetchInfoSaga({ payload }: Action<FetchInfoPayload>) {
   const { poll } = payload
   let isCloudEnabled = false
-  let isCloudClaimed = false
+  let isAgentClaimed = false
+  let isCloudAvailable = false
+  let isACLKAvailable = false
 
   try {
     const registry = yield select(selectRegistry)
     const wasCloudAvailable = registry?.isCloudAvailable || false
```

Comment:
I can't see where this gets changed. What about the case where this is false the first time it is read, but then it becomes true and then false again later?